src/utils/errors.ts

```diff
@@ -1,32 +1,32 @@
 import { StreamApiError } from 'getstream';
 
-export const handleError = (error, type, detail) => {
+export const handleError = (error: Error, type: string, detail: { kind?: string }) => {
```

Comment:
```suggestion
export const handleError = (error: Error, type: string, detail: Detail) => {
```
Move this one to its own type since it's used a bunch of times here.

Reply:
good point